src/node/suggestWinnerMoves.js

```diff
@@ -0,0 +1,42 @@
+const constants = require('./constants.js');
+const Point = require('./point.js');
+const checkWin = require('./checkWin.js');
+
+function suggestWinnerMoves(player, board) {
+  const array = createArrayWithWinnerMoves(player, board);
+  let suggestion = 'Movimentos ';
+  array.forEach(point => {
+    suggestion += `(${point.row}, ${point.col}) `;
+  });
+  suggestion += 'ganham o jogo!';
+  console.log(suggestion);
+}
+
+function createArrayWithWinnerMoves(player, board) {
+  const array = [];
+  for (let row = 0; row < constants.boardSize; row += 1) {
+    for (let col = 0; col < constants.boardSize; col += 1) {
+      const move = Point(row, col);
+      if (isWinnerMove(move, player, board)) {
+        array.push(move);
+      }
+    }
+  }
+  return array;
+}
+
+function isWinnerMove(point, player, board) {
+  if (board.tileIsOccupied(point)) {
+    return false;
+  }
+
+  board.placeSymbol(point, player.symbol);
+  if (checkWin(point, board)) {
+    board.eraseSymbol(point);
```

Comment:
I liked how you reused the `checkWin` function!
The main concern here would be the fact that you are changing the state of your game to do your calculations! 
you even created an erase symbol because you needed to "revert" the play, so this "suggestWinnerMoves" wouldn't create a side-effect on the game board. That's good, but even better, would be to avoid any side-effects.
you'll probably see this effect when your software grows larger, but having apparently harmless functions, manipulating the state of your game, is really hard to maintain.
You could achieve the same by doing:
``` js
function createArryaWithWinnerMovers(board) {
   for (let row = 0; row < constants.boardSize; row += 1) {
    for (let col = 0; col < constants.boardSize; col += 1) {
      const move = Point(row, col);
      if (isWinnerMove(move, player,  cloneBoard(board))) {
        array.push(move);
      }
    }
  }
}
```

In isWinnerMove you wouldn't have to revert to original state, because every time it iterates over the new move, it is passing a clean board.

Reply:
Now isWinnerMove is mutating but in a more controlled scenario!
One point of attention is that cloning object is an expensive operation, and usually delegated to libraries which implemented them in a more performatic way.

example of cloneMatrix which works only on bidimensional matrix:
``` js
const cloneMatrix = (matrix) => {
  const clonedMatrix = [];
  for(let row = 0; row< matrix.length; row += 1) {
     clonedMatrix.push([...matrix[row]]); // This operation is "creating a new array", and **spreading** the contents of another inside it. See Spreading for more
  }

  return clonedMatrix;
}
```